### PR TITLE
Enabling the Card/Paper elevation property.

### PR DESCRIPTION
**Pull request by SalahAdDin**
Sometimes we don't want to see cards with elevation, we just want to see them as plain text.

### CODE REVIEW

File: src/lib/TimelineItem/TimelineItem.js [hunk 1]
```diff
@@ -200,7 +200,7 @@ const TimelineItem = props => {
 					</div>
 				</div>
 				<Box className={props.direction === directions.LEFT ? classes.ArrowLeft : classes.ArrowRight}>
-					<Card component='div' className={classes.TimelineCard}>
+					<Card elevation={props.elevation} component='div' className={classes.TimelineCard}>
```
Comment (pkanellidis) [inline src/lib/TimelineItem/TimelineItem.js]: @SalahAdDin The elevation should be checked if it exists and if it doesn't it should be set to the default elevation value as per the MaterialUI libraries

**Comment by pkanellidis** [comment]
@SalahAdDin The elevation property should added in TimelineItem.d.ts file in the interface in order to get intellisense for the property.

File: src/lib/TimelineItem/TimelineItem.js [hunk 2]
```diff
@@ -231,7 +231,7 @@ const TimelineItem = props => {
...
```
Comment (pkanellidis) [inline src/lib/TimelineItem/TimelineItem.js]: @SalahAdDin Same here

**Comment by pkanellidis** [comment]
@SalahAdDin The elevation property should added in TimelineItem.d.ts file in the interface in order to get intellisense for the property.

File: src/lib/TimelineItem/TimelineItem.js (hunk 1 above)
Comment (SalahAdDin) [reply src/lib/TimelineItem/TimelineItem.js]: I don't know why, but if ew validate this property it does no work, but if you don't validate it, it works, also, when we don't put the property, the value would be the same as not raise: 1.

File: src/lib/TimelineItem/TimelineItem.js (hunk 2 above)
Comment (SalahAdDin) [reply src/lib/TimelineItem/TimelineItem.js]: I don't know why, but if ew validate this property it does no work, but if you don't validate it, it works, also, when we don't put the property, the value would be the same as not raise: 1.

File: src/lib/TimelineItem/TimelineItem.js (hunk 1 above)
Comment (pkanellidis) [reply src/lib/TimelineItem/TimelineItem.js]: Sorry these days have been a bit busy and I can't test it right now

**Pull request closed without merging**